test/e2e/provisioning/all_e2e_test.go

```diff
@@ -243,6 +244,30 @@ func TestPacketProvisioningE2E(t *testing.T) {
 	runScenarios(t, excludeSelector, params, PacketManifest, fmt.Sprintf("packet-%s", *testRunIdentifier))
 }
 
+func TestAlibabaProvisioningE2E(t *testing.T) {
+	t.Parallel()
+
+	// test data
+	accessKeyID := os.Getenv("ALIBABA_ACCESS_KEY_ID")
+	if len(accessKeyID) == 0 {
+		t.Fatal("unable to run the test suite, ALIBABA_ACCESS_KEY_ID environment variable cannot be empty")
+	}
+
+	accessKeySecret := os.Getenv("ALIBABA_ACCESS_KEY_SECRET")
+	if len(accessKeySecret) == 0 {
+		t.Fatal("unable to run the test suite, ALIBABA_ACCESS_KEY_SECRET environment variable cannot be empty")
+	}
+
+	excludeSelector := &scenarioSelector{osName: []string{"coreos"}}
```

Comment:
If we don't support this, we must add it to the validation

Reply:
Addressed here:
https://github.com/kubermatic/machine-controller/pull/643/files?file-filters%5B%5D=.go#diff-e96854fadd3b15014f29f87ac940a01bR421